frontend/sozisel-app/src/components/ParticipantActiveSession/ParticipantActiveSession.tsx

```diff
@@ -81,39 +51,99 @@ export default function ParticipantActiveSession({
-            />
-          )}
+  useEffect(() => {
+    if (eventData?.eventLaunched) {
+      setActiveEvent(eventData.eventLaunched);
+    }
+  }, [eventData]);
+
+  useEffect(() => {
+    if (sessionNotifications?.sessionNotifications) {
+      switch (sessionNotifications.sessionNotifications.info) {
+        case SessionInfo.SessionEnd:
+          history.push(`/`);
+          break;
+        default:
+          break;
+      }
+    }
+  }, [sessionNotifications, history]);
+
+  if (sessionLoading) {
+    return (
+      <>
+        <BasicNavbar />
+        <div className="ParticipantActiveSession">
+          <CircularProgress />
         </div>
-        <div className="moduleComponent">
-          {activeEvent && (
-            <ParticipantQuizContextProvider>
-              <ParticipantQuizEvent
-                onQuizFinished={() => setActiveEvent(null)}
-                token={token}
-                event={activeEvent}
+      </>
+    );
+  }
+
+  if (session?.sessionThumbnail) {
+    return (
+      <>
+        <BasicNavbar />
+        <div className="ParticipantActiveSession">
+          {!activeEvent && (
+            <div className="agendaComponent">
+              <ActiveSessionAgenda
+                agendaEntries={session.sessionThumbnail.agendaEntries}
+                estimatedTimeInSeconds={session.sessionThumbnail.estimatedTime}
+                sessionStartDate={new Date(session.sessionThumbnail.startTime)}
               />
-            </ParticipantQuizContextProvider>
+            </div>
+          )}
+          {session.sessionThumbnail.useJitsi && (
+            <div className="jitsi">
+              {!loading && data?.generateJitsiToken.token && (
+                <JitsiFrame
+                  roomId={id}
+                  token={data.generateJitsiToken.token}
+                  displayName={data.generateJitsiToken.displayName}
+                />
+              )}
+            </div>
+          )}
+          {!activeEvent && (
+            <div className="moduleComponent">
+              <ParticipantsList sessionId={id} />
+            </div>
           )}
+          {activeEvent &&
+            activeEvent.eventData.__typename === "ParticipantQuiz" && (
+              <div className="moduleComponent">
+                <ParticipantQuizContextProvider>
+                  <ParticipantQuizEvent
+                    onQuizFinished={() => setActiveEvent(null)}
+                    token={token}
+                    event={activeEvent}
+                  />
+                </ParticipantQuizContextProvider>
+              </div>
+            )}
         </div>
+        <Fab
+          variant="extended"
+          className="ParticipantSessionFab"
+          color="primary"
+          style={{ position: "fixed" }}
+          onClick={() => {
+            history.push(`/`);
+          }}
+        >
+          {t("components.ParticipantActiveSession.exitSession") ?? ""}
```

Comment:
?? is redundant, isn't it?

Reply:
![image](https://user-images.githubusercontent.com/48885911/121779094-43edeb80-cb9a-11eb-9dd6-a2760cb1e961.png)
seems that it isn't redundant

Reply:
kk